### PR TITLE
875042: Volume 1 UG documentation in Flutter PDF.

### PR DESCRIPTION
**Pull request by Sri-hari-haran-g**
Hi team,

Here we added volume 1 flutter features and bugs for UG documentation.

Regards,
Srihariharan G.

### CODE REVIEW

File: Flutter/pdf/working-with-annotations.md [hunk 1]
```diff
@@ -680,4 +680,114 @@ File('output.pdf').writeAsBytesSync(await document.save());
...
+
+{% endhighlight %}
+
+## Exporting annotations to JSON file
+
+To export annotation data to the JSON file from PDF document, you can use the exportAnnotation method in PdfDocument class.
+
+{% highlight dart %}
+
+    //Load an existing PDF document
```
Comment (M-A32) [inline Flutter/pdf/working-with-annotations.md:783]: //Load an existing PDF document.

File: Flutter/pdf/working-with-annotations.md [hunk 2]
```diff
@@ -680,4 +680,114 @@ File('output.pdf').writeAsBytesSync(await document.save());
...
+    PdfDocument document =
+        PdfDocument(inputBytes: File('input.pdf').readAsBytesSync());
+    //Export the annotations to JSON file format
```
Comment (M-A32) [inline Flutter/pdf/working-with-annotations.md:786]: //Export the annotations to JSON file format.

File: Flutter/pdf/working-with-annotations.md [hunk 3]
```diff
@@ -680,4 +680,114 @@ File('output.pdf').writeAsBytesSync(await document.save());
...
+    List<int> bytes = document.exportAnnotation(PdfAnnotationDataFormat.json);
+    //Save the FDF file
```
Comment (M-A32) [inline Flutter/pdf/working-with-annotations.md:788]: //Save the FDF file.

File: Flutter/pdf/working-with-annotations.md [hunk 4]
```diff
@@ -680,4 +680,114 @@ File('output.pdf').writeAsBytesSync(await document.save());
...
+    File('export.json').writeAsBytesSync(bytes);
+    //Dispose the document
```
Comment (M-A32) [inline Flutter/pdf/working-with-annotations.md:790]: //Dispose the document.

File: Flutter/pdf/working-with-annotations.md [hunk 5]
```diff
@@ -680,4 +680,114 @@ File('output.pdf').writeAsBytesSync(await document.save());
...
+{% highlight dart %}
+
+    //Load an existing PDF document
+    PdfDocument document =
+        PdfDocument(inputBytes: File('input.pdf').readAsBytesSync());
+    //Export the annotations to FDF file format
+    List<int> bytes = document.exportAnnotation(PdfAnnotationDataFormat.fdf);
+    //Save the FDF file
+    File('export.fdf').writeAsBytesSync(bytes);
+    //Dispose the document
```
Comment (M-A32) [inline Flutter/pdf/working-with-annotations.md:754]: //Dispose the document.

File: Flutter/pdf/working-with-annotations.md [hunk 6]
```diff
@@ -680,4 +680,114 @@ File('output.pdf').writeAsBytesSync(await document.save());
...
+To export annotation data to the FDF file from PDF document, you can use the exportAnnotation method in PdfDocument class.
+
+{% highlight dart %}
+
+    //Load an existing PDF document
+    PdfDocument document =
+        PdfDocument(inputBytes: File('input.pdf').readAsBytesSync());
+    //Export the annotations to FDF file format
+    List<int> bytes = document.exportAnnotation(PdfAnnotationDataFormat.fdf);
+    //Save the FDF file
```
Comment (M-A32) [inline Flutter/pdf/working-with-annotations.md:752]: //Save the FDF file.

File: Flutter/pdf/working-with-annotations.md [hunk 7]
```diff
@@ -680,4 +680,114 @@ File('output.pdf').writeAsBytesSync(await document.save());
...
+
+{% endhighlight %}
+
+## Exporting annotations to FDF file
+
+To export annotation data to the FDF file from PDF document, you can use the exportAnnotation method in PdfDocument class.
+
+{% highlight dart %}
+
+    //Load an existing PDF document
```
Comment (M-A32) [inline Flutter/pdf/working-with-annotations.md:747]: //Load an existing PDF document.

File: Flutter/pdf/working-with-annotations.md [hunk 8]
```diff
@@ -680,4 +680,114 @@ File('output.pdf').writeAsBytesSync(await document.save());
...
+    //Save the PDF document.
+    File('output.pdf').writeAsBytesSync(await document.save());
+    //Dispose the document.
+    document.dispose();
+
+{% endhighlight %}
+
+## Exporting annotations to FDF file
+
+To export annotation data to the FDF file from PDF document, you can use the exportAnnotation method in PdfDocument class.
```
Comment (M-A32) [inline Flutter/pdf/working-with-annotations.md:743]: To export annotation data to the FDF file from the PDF document, you can use the exportAnnotation method in PdfDocument class.

File: Flutter/pdf/working-with-annotations.md [hunk 9]
```diff
@@ -680,4 +680,114 @@ File('output.pdf').writeAsBytesSync(await document.save());
...
+    //Save the PDF document.
+    File('output.pdf').writeAsBytesSync(await document.save());
+    //Dispose the document.
+    document.dispose();
+
+{% endhighlight %}
+
+## Importing annotations from JSON file
+
+JSON stands for JavaScript Object Notation. It is a collection of key or value pairs and it is used for serializing and transmitting the structured data over a network connection. You can import the annotation data from the JSON file to PDF using the importAnnotation method in PdfDocument class.
```
Comment (M-A32) [inline Flutter/pdf/working-with-annotations.md:724]: JSON stands for JavaScript Object Notation. It is a collection of key or value pairs, and it is used for serializing and transmitting the structured data over a network connection. You can import the annotation data from the JSON file to PDF using the importAnnotation method in PdfDocument class.

**Review by chinnumuniyappan — APPROVED** [review]

**Pull request merged**